Kudu.Core/Environment.cs

```diff
@@ -95,7 +96,17 @@ public class Environment : IEnvironment
             _siteExtensionSettingsPath = Path.Combine(SiteRootPath, Constants.SiteExtensionsCachePath);
             _diagnosticsPath = Path.Combine(SiteRootPath, Constants.DiagnosticsPath);
             _locksPath = Path.Combine(SiteRootPath, Constants.LocksPath);
-            _sshKeyPath = Path.Combine(rootPath, Constants.SSHKeyPath);
+
+            string siteName = System.Environment.GetEnvironmentVariable("WEBSITE_SITE_NAME");
+            if (OSDetecter.IsOnWindows() || rootPath.TrimEnd(new char[] { '/', '\\' }).EndsWith(siteName, StringComparison.OrdinalIgnoreCase))
```

Comment:
when run in container, there is code to map root (%HOME% environment variable) to /home.

since Kudu is an open source project, normally ROOT for a user is /home/{user}, e.g when i tested on a linux machine but not using container. 

i make it easier to test kudu on both our container environment and normal Linux environment.

let me know if you think this is over kill. or i will add more comment and keep it.